packages/fork-choice/src/forkChoice/forkChoice.ts

```diff
@@ -1198,6 +1199,42 @@ export class ForkChoice implements IForkChoice {
       () => this.fcStore.unrealizedJustified.balances
     );
   }
+
+  verifyForwardCheckpoint({epoch, root}: phase0.Checkpoint): boolean {
```

Comment:
Can you add a nice long JSDoc comment linking to specs and explaining what this is doing and why?